dotdrop/cfg_yaml.py

```diff
@@ -1003,6 +1007,9 @@ def _import_config(self, path):
         self.trans_w = self._merge_dict(self.trans_w, sub.trans_w)
         self._clear_profile_vars(sub.variables)
 
+        self.imported_configs.append(path)
+        self.imported_configs += sub.imported_configs
```

Comment:
This line here is why `imported_configs` is a public variable and not a private one (prefixed with `_`). pylint complains that if `imported_configs` is private this is an access error.

In object-oriented languages with property accessor support (C++, Kotlin, C#...) it's normally allowed for any object to access private fields of another instance of the same class. I'd personally keep `imported_configs` private and silence pylint.